README.md

```diff
@@ -77,7 +66,7 @@ dotnet test SDKs\Compute\Compute.Tests\Compute.Tests.csproj
 
 ### Project Structure
 
-In "SDKs\< RPName >", you will find projects for services that have already been implemented
+In "SDKs\< Service Name >", you will find projects for services that have already been implemented
```

Comment:
@Petermarcu the last I checked with @weshaggard the new directory structure is not going to be until we are done with Swagger to SDK work.

Reply:
What is the difference between RP Name and Service name? Is RP a concept that we expect external contributors to understand? I was trying to use simpler words, not change to the new structure.